packages/@aws-cdk/aws-elasticsearch/test/domain.test.ts

```diff
@@ -81,7 +83,7 @@ describe('log groups', () => {
         SEARCH_SLOW_LOGS: {
           CloudWatchLogsLogGroupArn: {
             'Fn::GetAtt': [
-              'SlowSearchLogsE00DC2E7',
+              cdkAssert.stringLike(`${logicalId}SlowSearchLogs*`),
```

Comment:
Assert the concrete logical id that contains the domain as well. (in all tests)

Reply:
@iliapolo - so we should not use matchers for things like this? I assume that we *want* the test to break if the logicaIId changes?

Reply:
@peterb154 exactly

Reply:
@iliapolo amended the commit with concrete logicalIds